src/main/java/wordle/view/OutputView.java

```diff
@@ -0,0 +1,50 @@
+package wordle.view;
+
+import static wordle.domain.Tile.GRAY;
+import static wordle.domain.Tile.GREEN;
+import static wordle.domain.Tile.YELLOW;
+
+import java.util.List;
+import wordle.domain.Result;
+import wordle.domain.Tile;
+import wordle.domain.Trial;
+
+public class OutputView {
+    private static final String INIT_MESSAGE = "WORDLE을 " + Trial.MAX_TRIAL  +"번 만에 맞춰 보세요.\n"
+            + "시도의 결과는 타일의 색 변화로 나타납니다.";
+    private static final String GRAY_VIEW = "⬜";
+    private static final String YELLOW_VIEW = "\uD83D\uDFE8";
+    private static final String GREEN_VIEW = "\uD83D\uDFE9";
+
+    private OutputView() {
+    }
+
+    public static void printStartMessage() {
+        System.out.println(INIT_MESSAGE);
+    }
+
+    public static void printResult(final List<Result> results) {
+        results.forEach(result -> printTile(result.getTiles()));
+    }
+
+    private static void printTile(final List<Tile> tiles) {
+        String view = "";
+        for (final Tile tile : tiles) {
+            if (tile == GRAY) {
+                view += GRAY_VIEW;
+            }
+            if (tile == GREEN) {
+                view += GREEN_VIEW;
+            }
+            if (tile == YELLOW) {
+                view += YELLOW_VIEW;
+            }
+        }
+        System.out.println(view);
+    }
```

Comment:
다음과 같이 map을 사용해서 1 depth로 해보는건 어때?

```suggestion
    private static final String NEXT_LINE = System.lineSeparator();
    private static final Map<Tile, String> TILE_VIEW = Map.of(GRAY, GRAY_VIEW, GREEN, GREEN_VIEW, YELLOW, YELLOW_VIEW);

    public static void printResults(final List<Result> results) {
        final String result = results.stream()
                .map(it -> generateResult(it.getTiles()))
                .collect(joining(NEXT_LINE));
        System.out.println(result);
    }

    private static String generateResult(final List<Tile> tiles) {
        return tiles.stream()
                .map(TILE_VIEW::get)
                .collect(joining());
    }
```

Reply:
이거 체스 폰 만들때 경계 쓰던 허브스킬....!

Reply:
꿀팁 감사합니다.. 이렇게 또 배워간다